Test/nuget.config

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
I believe this is in nuget by default.

Reply:
I removed that entry and it looks that it works.